tests/by-util/test_cp.rs

```diff
@@ -5574,6 +5574,34 @@ fn test_preserve_attrs_overriding_2() {
     }
 }
 
+#[test]
+fn test_symlink_to_subdir() {
+    let (at, mut ucmd) = at_and_ucmd!();
+
+    at.touch("file");
+    at.mkdir("dir");
+
+    ucmd.args(&["--symbolic", "file", "dir"])
+        .fails()
+        .no_stdout()
+        .stderr_is("cp: dir/file: can make relative symbolic links only in current directory\n");
```

Comment:
This fails on Windows, because there it generates the error message with a backslash.

Reply:
Oh I see! Really appreciate the review. About the broken test in Windows, I think ``stderr_contains()`` can be used instead of ``stderr_is()`` to address this, I'm going to try it to solve this compatibility issue with Windows, along with the conflicts.